sdk/go/kubernetes/helm/v2/chart.go

```diff
@@ -191,6 +203,35 @@ func yamlDecode(ctx *pulumi.Context, text, namespace string) ([]map[string]inter
 	return ret.Result, nil
 }
 
+func isHelmV3() (bool, error) {
+
+	/*
+	   Helm v2 returns version like this:
+	   Client: v2.16.7+g5f2584f
+	   Helm v3 returns a version like this:
+	   v3.1.2+gd878d4d
+	   We can reasonably assume helm v2 if the version starts with Client
```

Comment:
This doesn't match the current regex

Reply:
Can you elaborate? I tested it pretty extensively but I could have missed something

Reply:
The regex now checks for `v3`, rather than the earlier behavior where it checked for `^Client`

Reply:
oh! Yes I'll update the comment